app/models/invenio_rdm_record_converter.rb

```diff
@@ -56,7 +58,7 @@ def initialize(generic_file=nil, collection_store={})
     @record = record_for_export
     @file = file_info
     @extras = extra_data
-    @prism_communities = map_collections_to_communities
+    @prism_communities = dh_collection_to_prism_community_collections
```

Comment:
given comment below:

```suggestion
    @prism_community = dh_collection_to_prism_community_collection
```

Reply:
Oh jeez, nice catch.